### ISSUE TITLE
phylophlan.py: error: unrecognized arguments: -u {params.unn} 

### ISSUE THREAD

**Issue by Arunkanth5** [WatsonLab/MAGpy#29]
[phylophlan.log](https://github.com/WatsonLab/MAGpy/files/5975189/phylophlan.log)
[phylophlan_help.txt](https://github.com/WatsonLab/MAGpy/files/5975200/phylophlan_help.txt)

I am testing MAGpy with conda env and getting "phylophlan.py: error: unrecognized arguments: -u {params.unn}". 

given bitbuket link was broken (https://bitbucket.org/nsegata/phylophlan), so i have tried with github repo and bioconda package for phylophlan. 
Both are same versions and giving same error. attached command help and error output log. 
Please check and suggest me correct versions to fix the issue. 

phylophlan {PhyloPhlAn version 3.0.60 (27 November 2020)} 

Git Repo : (https://github.com/biobakery/phylophlan)
(conda install -c bioconda phylophlan).

**Comment by mw55309** [comment]
Hello!

Sorry at the moment PhyloPhlAn integration is broken. MAGy worked with v 2.0 which is no longer available, and v 3 is a completely different piece of software, so it is not an easy swap.

To run MAGpy without PhyloPhlAn change:

```sh
rule all:
	 #input: "sourmash_report.csv", "diamond_bin_report_plus.tsv", expand("pfam/{sample}.pfam", sample=IDS), expand("sourmash/{sample}.sm", sample=IDS), expand("diamond_report/bin.{sample}.tsv", sample=IDS), "checkm_plus.txt", "tree/MAGpy/MAGpy.tree.nwk"
	 input: "sourmash_report.csv", "diamond_bin_report_plus.tsv", expand("pfam/{sample}.pfam", sample=IDS), expand("sourmash/{sample}.sm", sample=IDS), expand("diamond_report/bin.{sample}.tsv", sample=IDS), "checkm_plus.txt"
```

Sorry!

**Comment by Arunkanth5** [comment]
tried as you sugged and its working with out phylophlan funtionality. 

is there any plan to release fixed version near soon ??

**Comment by mw55309** [comment]
Yes! Though note PhyloPhlAn 2.0 is no longer available, only PhyloPhlAn 3.0, which is a *very* different piece of software, and doesn't have the same functionality :(

**Comment by mw55309** [comment]
Hey

There is a new release today that fixes a lot of sourmash and phylophlan bugs

Please test it!

Cheers
Mick
